itscalledsoccer/client.py

```diff
@@ -46,11 +47,22 @@ def __init__(
 
         self.session = CACHE_SESSION
         self.base_url = self.BASE_URL
-        self.players = self._get_entity("player")
-        self.teams = self._get_entity("team")
-        self.stadia = self._get_entity("stadia")
-        self.managers = self._get_entity("manager")
-        self.referees = self._get_entity("referee")
+        self.lazy_load = lazy_load
+
+        if self.lazy_load:
+            print("Lazy loading enabled. Initializing client without entity data.")
```

Comment:
```suggestion
            self.LOGGER.info("Lazy loading enabled. Initializing client without entity data.")
```
We should use the `LOGGER` instead of `print`